paw-webapp/webapp/src/main/java/ar/edu/itba/paw/webapp/mail/MailServiceImpl.java

```diff
@@ -52,9 +52,9 @@ public void prepare(MimeMessage mimeMessage) throws Exception {
                         new InternetAddress(user.getEmail()));
                 mimeMessage.setText("Dear "
                         + user.getUsername()
-                        + ", \nYour new password is: "
-                        + newPassword
-                        + "\nPlease log in with your new password, and if you wish, head to your profile page and change your password to something easier to remember.");
+                        + ", \nThis email was sent to you because someone requested a password reset on your account. \n\n Visit the following URL to set a new password: "
```

Comment:
`someone` queda re mal, es como que un flaco cualquiera vino y pidió que te reseteen la cuenta (más allá de que se pueda hacer o no en nuestra página, queda mal decirlo así). Yo pondría algo como `This email was sent to you after a request to reset your account password.  Visit...`